src/olympia/blocklist/migrations/0038_remove_blockversion_soft.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.16 on 2024-10-31 13:26
+
+from django.db import migrations
+import olympia.amo.fields
+
+
+def convert_block_type_to_soft(apps, schema_editor):
```

Comment:
Wouldn't we want the opposite? to convert any remaining rows with soft populated and block_type NOT populated to use block_type, before dropping the column? What's the point of populating soft if we are just going to drop it?

Reply:
because if you see line 22, this code runs when rolling back this migration.

Reply:
Then we should also include another method to backfill soft to block_type no?

Reply:
For context T4 in [this timeline](https://github.com/mozilla/addons-server/pull/22811#discussion_r1825753311) is why I think we need that back fill.